Update readme file ,create html and css files, integrate linters, upload icons, created .gitignore file, applied box model, corrected css file

evillaz opened the pull request:
This branch's purpose was to:
Customize the readme file for the current project.
Create HTML structure and content for the project.
Apply styling through a CSS file to enrich the project.
Integrate linters to flag patterns that could cause errors or similar problems.
Upload icons SVG files to enrich the project's styling.
Created a .gitignore file to follow the best practices on a professional Github repo
////
Changed name from HTML file
Corrected CSS file and applied box model to the project

evillaz committed:
update readme

customize readme file for the project

evillaz committed:
add logo file

 add logo image to readme

evillaz committed:
Create linters.yml file

Integrate linters into the project

evillaz committed:
Create My-Portfolio.html file

Create the HTML structure and content for the project

evillaz committed:
Upload icon files

Upload icon files to enrich the HTML project

evillaz committed:
Upload icon files

Upload icon files to enrich the HTML project

evillaz committed:
Delete Icon - Menu.svg

evillaz committed:
Delete logo.svg

evillaz committed:
Upload icon file

Upload icon from the menu section

evillaz committed:
Create style.css file

Add styling for the HTML project

evillaz committed:
Create .gitignore file

Add a .gitignore file following the best practices for a professional Github repo

evillaz commented:
hi fellow reviewer I'm taking care of the style.css that linters flag out

evillaz committed:
upload style.css

upload modifications to style.css file

evillaz committed:
Update style.css

upload modifications to style.css

bobb-Rob commented on My-Portfolio.html:
```diff
@@ -0,0 +1,41 @@
+      <header id="header">
+        <div class="flex-container">
+          <nav id="nav-bar">
+            <div id="logo-placeholder">
+              <h2>John Doe</h2>
+            </div>
```
- [x] [**REQUIRED**] Usually, 99.99% of websites, the logo will be a link to the home page. Therefore, I think you should use the `anchor tag` to wrap this logo. I suggest that you should replace the `h2` tag with an `a` tag. with `href` to the `index.html`. This is essentially for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.

bobb-Rob commented on My-Portfolio.html:
```diff
@@ -0,0 +1,41 @@
+            <img id="logo4" src="icons/logo4.svg" alt="logo4" />
+            <img id="logo5" src="icons/logo5.svg" alt="logo5" />
+          </div>
+          <img id="background-decoration" src="icons/Illustration.svg" alt="decoration" />
```
- [x] [**REQUIRED**] This image is a background image for the headline section and should not be added to the screen with an `img` tag, but with CSS `background-image` property. There is a rule of thumb, if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

bobb-Rob commented on css/style.css:
```diff
@@ -0,0 +1,166 @@
+
+nav {
+  height: 48px;
+  width: 375px;
```
- [x] I notice that you are using a fixed width for most of your elements, Which is why your design is not responsive. I advise that you remove all the fixed widths `width: 375px;` you have in your design. Allow the design scale according to screen size. 

#### When I view your design in mobile view. It becomes very tiny.  
![image](https://user-images.githubusercontent.com/84156714/211929528-e915a3b6-caff-4f7b-bca1-fc32b00bcde7.png)

bobb-Rob commented on My-Portfolio.html:
```diff
@@ -0,0 +1,41 @@
+      </header>  
+      <div>
+        <section id="headline">
+          <p id="primary-text">Hey There. I'm Eduardo</p>
```
- [x] [REQUIRED] This primary text should be wrapped with an `h1` element instead of `p` tag because of SEO reasons. If you want that google (and other search engines) knows about a specific topic, in this case, you, use an `h1` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.

bobb-Rob commented on My-Portfolio.html:
```diff
@@ -0,0 +1,41 @@
+          <p id="primary-text">Hey There. I'm Eduardo</p>
+          <p id="supporting-text">I'am a Software Developer</p>
+          <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+          <div class="flex-container" id="social-media">
+            <img id="logo1" src="icons/logo1.svg" alt="logo1" />
+            <img id="logo2" src="icons/logo2.svg" alt="logo2" />
+            <img id="logo3" src="icons/logo3.svg" alt="logo3" />
+            <img id="logo4" src="icons/logo4.svg" alt="logo4" />
+            <img id="logo5" src="icons/logo5.svg" alt="logo5" />
+          </div>
```
- [x] [REQUIRED] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag). 
* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.
* Kindly fix this.

bobb-Rob commented on css/style.css:
```diff
@@ -0,0 +1,166 @@
+}
+
+#primary-text {
+  left: 24px;
+  top: 120px;
+  border-radius: nullpx;
+  width: 327px;
+  height: 124px;
+  margin: 0;
+  position: absolute;
```
- [x] I think that if you use Box model _(i.e your elements, their boundaries, and space both in and outside the target element.)_, you will have your elements flow naturally and no need for `position: absolute`. Kindly read about [CSS Box Model here.](https://blog.hubspot.com/website/css-box-model).

bobb-Rob reviewed (changes requested):
## STATUS: CHANGES REQUIRED ♻️
Hi @evillaz , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)

## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

evillaz committed:
change name index.html, update html and css file

change name from html file to index.html, updated html and css file with the corrections

evillaz committed:
update css file with corrections

linters corrections

evillaz committed:
Update style.css

evillaz committed:
update logos links

linked every logo with social media website

AHP15 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+        <div class="flex-container">
+          <nav id="nav-bar">
+            <div id="logo-placeholder">
+              <a href="index.html">John Doe</a>
```
[OPTIONAL] Kindly change the name here to your real name as the website is about you

AHP15 commented on index.html:
```diff
@@ -0,0 +1,66 @@
+            <ul id="social-media">
+              <li>
+                <a href="https://github.com/evillaz">            
+                  <img id="logo1" src="icons/logo1.svg" alt="logo1" />
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/eduardo-villarreal-144a8925a/">
+                  <img id="logo2" src="icons/logo2.svg" alt="logo2" />
+                </a>
+              </li>
+              <li>
+                <a href="https://angel.co/u/eduardo-villarreal-3">
+                  <img id="logo3" src="icons/logo3.svg" alt="logo3" />
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/evillazz">
+                  <img id="logo4" src="icons/logo4.svg" alt="logo4" />
+                </a>
+              </li>
+              <li>
+                <a href="https://medium.com/@evillaz123">
+                  <img id="logo5" src="icons/logo5.svg" alt="logo5" />
```
[OPTIONAL] I highly suggest that you make the `alt` attributes more descriptive because it will increase your website accessibility (When a user is using a screen reader it read the `alt` value whenever it encounters an `img` tag check [this](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt) for more info)

AHP15 reviewed (approved):
Hi @evillaz , 

# STATUS: APPROVED 🥇 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![](https://user-images.githubusercontent.com/69011963/151541305-67f4f8e4-c79e-4f9d-a6cf-2c7efc3102c6.gif)

## Highlights
✅ No linter errors
✅ Good commit messages
✅ You did a great job implementing all the required changes
## Optional suggestions
- Check the comments below
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

evillaz committed:
update index.html

correcting html file for merging

evillaz committed:
Update index.html

correcting html file for merging

evillaz commented:
It took me a while to solve but I finally managed to apply the corrections
for this review. Already worked on your optional changes and made them as
well. Thanks for your time reviewing.

On Thu, Jan 12, 2023 at 3:52 AM Abdessittir Harkati <
***@***.***> wrote:

> ***@***.**** approved this pull request.
>
> Hi @evillaz <https://github.com/evillaz> ,
> STATUS: APPROVED 🥇
>
> Your project is complete! There is nothing else to say other than... it's
> time to merge it [image: :shipit:]
> Congratulations! 🎉
>
>
> <https://user-images.githubusercontent.com/69011963/151541305-67f4f8e4-c79e-4f9d-a6cf-2c7efc3102c6.gif>
> Highlights
>
> ✅ No linter errors
> ✅ Good commit messages
> ✅ You did a great job implementing all the required changes
> Optional suggestions
>
>    - Check the comments below
>    *Every comment with the [OPTIONAL] prefix won't stop the approval of
>    this PR. However, I strongly recommend you to take them into account as
>    they can make your code better. Some of them were simply missed by the
>    previous reviewer and addressing them will really improve your application.*
>
> Cheers and Happy coding!👏👏👏
>
> Feel free to leave any questions or comments in the PR thread if something
> is not 100% clear.
> Please, remember to tag me in your question so I can receive the
> notification.
> ------------------------------
>
> *As described in the Code reviews limits policy
> <https://microverse.zendesk.com/hc/en-us/articles/1500004088561> you have a
> limited number of reviews per project (check the exact number in your
> Dashboard). If you think that the code review was not fair, you can request
> a second opinion using this form <https://airtable.com/shrQAqnBwek5a0O0s>.*
> ------------------------------
>
> In index.html
> <https://github.com/evillaz/my-portfolio/pull/1#discussion_r1067844615>:
>
> > +<html>
>
> +  <head>
>
> +    <meta charset="UTF-8">
>
> +    <title>My portfolio</title>
>
> +    <link rel="stylesheet" href="css/style.css">
>
> +    <link ***@***.***&display=swap" rel="stylesheet">
>
> +    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
>
> +    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
>
> +  </head>
>
> +  <body>
>
> +    <div class="container">
>
> +      <header id="header">
>
> +        <div class="flex-container">
>
> +          <nav id="nav-bar">
>
> +            <div id="logo-placeholder">
>
> +              <a href="index.html">John Doe</a>
>
>
> [OPTIONAL] Kindly change the name here to your real name as the website is
> about you
> ------------------------------
>
> In index.html
> <https://github.com/evillaz/my-portfolio/pull/1#discussion_r1067849530>:
>
> > +                  <img id="logo1" src="icons/logo1.svg" alt="logo1" />
>
> +                </a>
>
> +              </li>
>
> +              <li>
>
> +                <a href="https://www.linkedin.com/in/eduardo-villarreal-144a8925a/">
>
> +                  <img id="logo2" src="icons/logo2.svg" alt="logo2" />
>
> +                </a>
>
> +              </li>
>
> +              <li>
>
> +                <a href="https://angel.co/u/eduardo-villarreal-3">
>
> +                  <img id="logo3" src="icons/logo3.svg" alt="logo3" />
>
> +                </a>
>
> +              </li>
>
> +              <li>
>
> +                <a href="https://twitter.com/evillazz">
>
> +                  <img id="logo4" src="icons/logo4.svg" alt="logo4" />
>
> +                </a>
>
> +              </li>
>
> +              <li>
>
> +                <a ***@***.***">
>
> +                  <img id="logo5" src="icons/logo5.svg" alt="logo5" />
>
>
> [OPTIONAL] I highly suggest that you make the alt attributes more
> descriptive because it will increase your website accessibility (When a
> user is using a screen reader it read the alt value whenever it
> encounters an img tag check this
> <https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt>
> for more info)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/evillaz/my-portfolio/pull/1#pullrequestreview-1245174994>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A4TELPILNBQ43TZP6YBMRX3WR7A5BANCNFSM6AAAAAATYP5JVM>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
